server/controllers/homepageController.ts

```diff
@@ -52,4 +71,391 @@ export default class HomepageController {
       return res.redirect(`${config.serviceUrls.digitalPrisons}/global-search/results?searchText=${name}`)
     }
   }
+
+  getServices(req: CustomRequest, res: Response, next: NextFunction) {
+    const { whereaboutsMaintenanceMode } = config.app
+    const { keyworkerMaintenanceMode } = config.app
+
+    let whereaboutsConfig
+    if (whereaboutsMaintenanceMode) {
+      whereaboutsConfig = { enabled: false }
+    } else {
+      whereaboutsConfig = { enabled: true }
```

Comment:
DPS has calls to APIs to determine `whereaboutsConfig` and `keyworkerPrisonStatus` - do we not need these as well?